index.bs

```diff
@@ -839,6 +841,27 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  If <code>|credentialCreationData|.[=AttestationConveyancePreferenceOption=]</code>'s value is
+                    <dl class="switch">
+                        :   "none"
+                        ::  replace potentially uniquely identifying information (such as [=AAGUID=] and 
+                            [=attestation certificates=]) in the [=attested credential data=] and [=attestation statement=],
+                            respectively, with blinded versions of the same data.
+                            
+                        Issue: need to specify how to perform such blinding?
+    
+                        :   "indirect"
+                        ::  The client MAY replace the [=AAGUID=] and [=attestation statement=] with a more privacy-friendly 
+                            and/or more easily verifiable version of the same data (for example, by employing a [=Privacy CA=]).
+    
+                        :   "direct"
+                        ::  convey the authenticator's [=AAGUID=] and [=attestation statement=], unaltered, to the RP.
+  
+                        Issue: @balfanz wishes to add to the "direct" case: 
+                            If, for whatever reason, the client cannot pass on the authenticator-generated attestation
+                            statement, it MUST terminate the credential generation operation with a "NotAllowedError".
```

Comment:
This needs to be resolved for interop, but ... I think I am OK with resolving it for CR, too.

If we get agreement on this PR, let's follow up and add an issue number into this line.

Reply:
I am concerned about the phrase "for whatever reason, the client cannot pass", since it is pretty vague.
Would not installing+activating some "enterprise policy" be such a "reason"?
I would be fine with 
a) leaving the statement as is, i.e. "convey the authenticator's AAGUID and attestation statement unaltered to the RP." or
b) defining such reasons being "if the authenticator's attestation statement violates the privacy principles, ...", where privacy principles refer to the minimum number of 100k Authenticator of the same model or not publishing Metadata Statement regarding this authenticator.

Reply:
Responding in two places to make sure this is not lost:

Agree with @rlin1 concern. I'd like to suggest the following re-wording:

`If authenticator violates privacy requirements of the attestation scheme it is using, the client MUST terminate the credential generation operation with a AttestationNotPrivateError' 

In my opinion, this wording scopes the problem to privacy and at the same time extends it beyond batch attestation issues. For example, improper implementation of the TCG attestation scheme could also be covered.

Reply:
Note that @balfanz has updated this language in PR #636 with this commit: 825cce7